Slider/Assets/Scenes/Village.unity

```diff
@@ -114137,27 +114264,32 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[7].dialogueChain.Array.data[2].dialogue
-      value: we might need your artifact to <wavy>fix the village</wavy>.
+      value: its all out of whack... now how do we <wavy>fix the village</wavy>?
```

Comment:
i might revert this line its mostly to indicate to the player that now is the time to do the 8 puzzle thing and put everything back